packages/pg-v5/index.js

```diff
@@ -52,7 +51,10 @@ exports.commands = flatten([
   require('./commands/unfollow'),
   require('./commands/upgrade'),
   require('./commands/vacuum_stats'),
-  require('./commands/wait')
+  require('./commands/wait'),
+
+  // must be last
```

Comment:
Can you give more context here? I'm curious what's up, but also concerned about this becoming ripe for cargo-culting even if the situation changes.

Reply:
I think it's a circular dependency, but I can't figure out where it is stemming from exactly

Reply:
I might also be able to fix this by putting `require('./lib/host')` at the top, let me try that